CheckersCheckpoint/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CheckersCheckpoint
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            new Game();
```

Comment:
It might look like clean code, but doing a bunch of work in a constructor is a bad idea. You should have a method called `Play` or something like that. This becomes a problem when someone (could be you or someone else or a framework) want to figure out how this class works and this constructor calls never returns. In the next class you will use an [DI](https://stackoverflow.com/questions/50718586/what-is-a-di-container) framework and if it runs into a class like this, your program will just hang and there is not an easy way to figure out what is happening.